### PR TITLE
API examples: move csaf_searcher to a lower prio place

### PR DESCRIPTION
**Pull request by s-l-teichmann**
The `csaf_searcher` (here renamed to `purls_searcher`) is not a top level tool of the csaf_distribution.
It it just a little API example and should be treated as such.
As the whole API infrastructure is WIP we should not suggest that we give
the  same grants to the examples as for the **real** tools.

### CODE REVIEW

**Review by JanHoefelmeyer — DISMISSED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -28,6 +25,12 @@ is a tool for testing a CSAF Trusted Provider according to [Section 7 of the CSA
 ### [csaf_aggregator](docs/csaf_aggregator.md)
 is a CSAF Aggregator, to list or mirror providers.
 
+## Other stuff
+
+### [examples](./examples/README.md)
+are small examples how to use `github.com/csaf-poc/csaf_distribution` as an API.
```
Comment (ThomasJunk) [inline README.md:31]: are small examples of how to use ...

File: README.md [hunk 2]
```diff
@@ -28,6 +25,12 @@ is a tool for testing a CSAF Trusted Provider according to [Section 7 of the CSA
...
+Currently this work in progress. They may be extended and/or changed in the future.
```
Comment (ThomasJunk) [inline README.md:32]: Currently this is work in progress. As they are subject to change they may be extended/altered in the future.

**Review by ThomasJunk — DISMISSED** [review]
LGTM (suggested minor wording changes)

**Review by ThomasJunk — APPROVED** [review]
LGTM

**Review by ThomasJunk — APPROVED** [review]
+1

**Pull request merged**